tests/lax_test.py

```diff
@@ -2284,6 +2284,22 @@ def test_select_jvp_complexity(self):
                                              (x,), (1.,)))(1.)
     self.assertLen(jaxpr.jaxpr.eqns, 2)
 
+  def testRngBitGenerator(self):
```

Comment:
You need to gate this test on having a sufficiently new jaxlib version. The C++ parts of jax are on a separate release schedule to the Python parts.

Perhaps
```
@unittest.skipIf(jax.lib.version < (0, 1, 62), "Needs jaxlib 0.1.62 or newer")
```
or similar?

Reply:
I had to condition not only the test but the logic in `lax.py` and in `lax/__init__.py` because `RandomAlgorithm = xops.RandomAlgorithm` requires the op to be present.

I am not too happy with the added complexity so it might be nicer if we wait with this until a new version of jaxlib gets released.

Reply:
I thought #5714 or #6001 which released jaxlib version 0.1.61 should have solved the issue but I might be still missing something.

Is there something I can do to make this work without having to add a version dependent import into `lax/__init__.py`?

Reply:
I don't think you need to gate the logic in `lax.py`. Remember that Python is dynamic, so if you don't execute `rng_bit_generator` it won't fail due to missing symbols.

If you want to be sure that the rng bit generator is present, we would need to bump the minimum jaxlib version. We should probably do that soon regardless, but I think gating the test (only) on a jaxlib version check should suffice to get this submitted.